public/app/features/explore/Logs/Logs.tsx

```diff
@@ -267,6 +267,7 @@ class UnthemedLogs extends PureComponent<Props, State> {
 
     reportInteraction('grafana_explore_logs_visualisation_changed', {
       newVisualizationType: visualisation,
+      datasourceType: this.props.datasourceType,
```

Comment:
I think we do this in most cases for feature tracking:

```suggestion
      datasourceType: this.props.datasourceType ?? 'unknown',
```